oss-test/src/main/java/dev/ocpd/oss/FileStoreTest.kt

```diff
@@ -0,0 +1,128 @@
+package dev.ocpd.oss
+
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.net.MalformedURLException
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.StandardCopyOption
+
+abstract class FileStoreTest {
+
+    companion object {
+        private const val TEST_FILE_KEY_PREFIX = "oss-test-images"
+        private const val TEST_FILE_KEY = "$TEST_FILE_KEY_PREFIX/binglogo.png"
+        private val TEST_FILE_URL: URL = try {
```

Comment:
We don't need to check for invalid URL here, I guess? This is a preset value and will never be invalidated.